tests/protodata-extension/src/main/java/io/spine/protodata/test/uuid/UuidJavaRenderer.java

```diff
@@ -70,6 +70,9 @@ public final class UuidJavaRenderer extends JavaRenderer {
      */
     @Override
     protected void render(SourceFileSet sources) {
+        if (!sources.sourceRoot.endsWith("java")) {
```

Comment:
@armiol, this line breaks the build sayin that the `sourceRoot` property is `private`. It was indeed until I opened it.

It looks like indication that we test against the "dev" version, and not the one under the development. I've tried to force versions in the root build, but it didn't give much. I still get the "private" property error.

Let's discuss when time permits.


I've opened the